e2e/utils/zetacore.go

```diff
@@ -20,7 +20,7 @@ const (
 	AdminPolicyName       = "admin"
 	OperationalPolicyName = "operational"
 
-	DefaultCctxTimeout = 4 * time.Minute
+	DefaultCctxTimeout = 6 * time.Minute
```

Comment:
I am not sure why its needed now , but incrementing this solved the issues I have been facing when running tests on my local network

Reply:
Would be good to track this as issue

Reply:
https://github.com/zeta-chain/node/issues/2690

Reply:
Could be related to the fact that we introduced a couple e2e tests which in turn invoke `MustWaitForTxReceipt`.